pom.xml

```diff
@@ -830,7 +830,7 @@
                 <dependency>
                     <groupId>mysql</groupId>
                     <artifactId>mysql-connector-java</artifactId>
-                    <version>8.0.28</version>
+                    <version>5.1.49</version>
```

Comment:
It doesn't match with the mySqlVersion property, please use the property here to avoid problems.